distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/SampleNsiliClient.java

```diff
@@ -146,6 +146,15 @@
   private static final Query STANDING_ALL_QUERY =
       new Query(NsiliConstants.NSIL_ALL_VIEW, "NSIL_CARD.identifier like '%'");
 
+  private static final String TAB = "\t {}";
```

Comment:
Might be useful to give this a more descriptive name like `TAB_LOG_MSG`

Reply:
Updated 👍